test/rhsmlib/dbus/test_register.py

```diff
@@ -28,7 +28,6 @@
 
 CONSUMER_CONTENT_JSON_SCA = """{"hypervisorId": null,
         "serviceLevel": "",
-        "autoheal": true,
```

Comment:
this is a sample answer from candlepin, which does include `autoheal`; hence, please leave it there

Reply:
Reverted.